platform/platform-app/src/main/resources/features.xml

```diff
@@ -28,6 +28,8 @@
           xmlns="http://karaf.apache.org/xmlns/features/v1.3.0"
           xsi:schemaLocation="http://karaf.apache.org/xmlns/features/v1.3.0 http://karaf.apache.org/xmlns/features/v1.3.0">
 
+    <repository>mvn:org.apache.karaf.features/standard/${karaf.version}/xml/features</repository>
+    <repository>mvn:org.ops4j.pax.cdi/pax-cdi-features/${pax.cdi.version}/xml/features</repository>
```

Comment:
It could just be adding in some features that aren't installed by default. I didn't want to remove it, since it is included today. Someone out there might be using those features.